portal/src/graphql/portal/CreateProjectScreen.tsx

```diff
@@ -14,13 +14,14 @@ import { useCreateAppMutation } from "./mutations/createAppMutation";
 import { useAppListQuery } from "./query/appListQuery";
 import { ErrorParseRule, makeReasonErrorParseRule } from "../../error/parse";
 import { useSimpleForm } from "../../hook/useSimpleForm";
+import { randomProjectName } from "../../util/projectname";
 
 interface FormState {
   appID: string;
 }
 
 const DEFAULT_STATE: FormState = {
-  appID: "",
+  appID: randomProjectName(),
```

Comment:
This is a module constant. So if I create an app without reloading the app, the second time I create the app, the app name must have been seen by me. That is not correct.